docs/hub/spaces-sdks-gradio.md

```diff
@@ -55,3 +55,13 @@ This Python script uses a [🤗 Transformers pipeline](https://huggingface.co/do
 <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-gradio.png"/>
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-hot-dog-gradio-dark.png"/>
 </div>
+
+## Embed Gradio Spaces on other webpages
+
+You can use the HTML `<iframe>` tag to embed a Gradio Space as an inline frame on other webpages. Simply include the name of your Space in the URL, delimited with a dash `"-"` between the user/org and Space name, followed by the `.hf.space` suffix. For example, the demo above can be embedded in these docs with the following tag:
```

Comment:
Actually iframes are not the encouraged approach. The encouraged approach is to use web components. With iframes you will face issues of having responsive embeds that look good in different screens. Maybe we can link to the official guide for this https://gradio.app/sharing_your_app/#embedding-hosted-spaces

Reply:
Thanks, I didn't know this! Fixed in [2015036](https://github.com/huggingface/hub-docs/pull/489/commits/2015036f206497319f4b5ed6e9d9e27b132d423b)